charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,80 @@
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+
+## Objectives
+
+The data processing service aims to develop scientific analysis pipelines and the infrastructure to run them reliably and at-scale for all analyses on all scientific data types in the Human Cell Atlas Data Coordination Platform. The team working on the service will also operate the infrastructure. Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community.
+
+## In-scope
+
+* Scoping or review of scoping for new pipelines.  
+* Creation of all standardized pipelines for scientific data.  
+* Review, on-boarding, and potential creation of community pipelines.  
+* Review of community tools for use in data processing pipelines.  
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks.  
+* Establishing standard benchmarking used to evaluate data processing pipelines.  
+* Creation, maintenance, and operation of the HCA DCP pipeline execution environment.  
+* Investigating and identifying data for pipeline benchmarking.  
+* Initiation and management of the execution of all data processing pipelines (community or standardized) that run in the HCA DCP.  
+* Evaluation and adoption of file formats (and associated standards) for outputs of pipelines.  
+* Establishing and operating automated practices around scientific and engineering testing of pipelines.  
+* Outreach to users on the usability of data in downstream applications.  
+* Outreach to developers on pipeline development in the HCA DCP.  
+* Suggestion but not selection of new standardized pipelines for the HCA DCP.  
+
+### Scientific "guardrails" [Optional]
+
+* Initial versions of standardized pipelines are reviewed by the AWG.  
+* Changes to standardized pipelines that affect the scientific output are reviewed by the AWG, engineering updates that do not affect the scientific output are not reviewed by the AWG.  
+* Selection of standardized pipelines to work on are approved by the HCA DCP Governance Group (this does not apply to community pipelines, community pipelines are on-boarded as a service to the scientific community). Prioritization of the work to develop selected pipelines is performed by the data processing service team.  
+
+## Out-of-scope
+
+* Educational training or user assistance on how to perform data analysis with outputs of pipelines.  
+* Recommendations or demonstrations of downstream tool use beyond initial data loading.  
+* Novel methodological development is not performed by the data processing team unless it is essential to the pipeline, it does not existent in community tools, and it represents minimal effort to implement.  
+* Software engineering 3rd party tools (although may happen in high value community tools, is not a requirement of the service and left to the discretion of the team).  
```

Comment:
I would leave out the caveat here otherwise people will view this statement as a negotiating position rather than something your team can decide for yourselves.

Reply:
Took your advice ;-)